source/style/_global/docsearch.less

```diff
@@ -1,5 +1,5 @@
 @color-highlight-header-background: @color-mediumlight;
-@color-highlight-text: @color-primary;
+@color-highlight-text: @color-text-highlight;
```

Comment:
Is it possible that these changes are going to break the Meteor search?  (They're in `_global`.)

Reply:
Nevermind.  Looks okay! 

![image](https://user-images.githubusercontent.com/841294/40797058-95f73a48-650f-11e8-8e67-367281f1037f.png)